### PR TITLE
fix a few typos of PEP 252, 253, 367, 3135

### PR DESCRIPTION
**Pull request by zhangyangyu**
Just as the title, while reading these PEPs, find a few typos, fix them.

### CODE REVIEW

File: pep-0252.txt [hunk 1]
```diff
@@ -192,16 +192,16 @@ Specification of the class-based introspection API
...
        the same name *may* override the dictionary item.)
 
     2. The __class__ attribute on regular objects
 
        A regular object usually has a __class__ attribute.  If it
        does, this references a meta-object.  A meta-object can define
-       static attributes for the regular object whose __class__ it
-       is.  This is normally done through the following mechanism:
+       static attributes for the regular object whose __class__ is
+       it.  This is normally done through the following mechanism:
```
Comment (Rosuav) [inline pep-0252.txt]: This one was originally correct.

**Comment by Rosuav** [comment]
One of those was already correct ("is it" vs "it is"). The others LGTM.

**Comment by Rosuav** [comment]
Can't merge just part of the commit; probably the easiest fix would be to amend your commit to remove that part, and force-push to your clone.

**Comment by zhangyangyu** [comment]
Remove the unnecessary modification.

**Comment by Rosuav** [comment]
Thanks for picking up those typos.

**Pull request merged**